### PR TITLE
translate: hugegraph-client.md into english

### PR DESCRIPTION
**Pull request by Alita-Nan**
Translate the quick start of client.

### CODE REVIEW

**Review by simon824 — APPROVED** [review]
LGTM

**Review by imbajin — APPROVED** [review]

**Comment by imbajin** [comment]
and the client have another api doc in [docs/client/hugegraph-client.md](https://github.com/Alita-Nan/incubator-hugegraph-doc/blob/translate/content/en/docs/clients/hugegraph-client.md)

I'll add the need in issue later

File: content/en/docs/quickstart/hugegraph-client.md [hunk 1]
```diff
@@ -4,37 +4,35 @@ linkTitle: "Develop with HugeGraph-Client"
...
 
-#### 4.1 新建Maven工程
+#### 4.1 Build New Maven Project
 
-可以选择Eclipse或者Intellij Idea创建工程：
+Using IDEA or Eclipse to create the project:
 
-- [Eclipse新建Maven工程](http://www.vogella.com/tutorials/EclipseMaven/article.html)
-- [Intellij Idea 创建maven工程](https://vaadin.com/docs/-/part/framework/getting-started/getting-started-idea.html)
+- [Build by eclipse](http://www.vogella.com/tutorials/EclipseMaven/article.html)
```
Comment (javeme) [inline content/en/docs/quickstart/hugegraph-client.md:32]: eclipse => Eclipse

File: content/en/docs/quickstart/hugegraph-client.md [hunk 2]
```diff
@@ -4,37 +4,35 @@ linkTitle: "Develop with HugeGraph-Client"
...
-### 3 使用流程
+### 3 How To Use
 
-使用HugeGraph-Client的基本步骤如下:
+The basic steps to use HugeGraph-Client are as follows:
 
-- 新建Eclipse/ IDEA Maven项目；
-- 在pom文件中添加HugeGraph-Client依赖；
-- 创建类，调用HugeGraph-Client接口；
+- Build a new maven project by IDEA or eclipse
```
Comment (javeme) [inline content/en/docs/quickstart/hugegraph-client.md:20]: maven => Maven
eclipse => Eclipse

File: content/en/docs/quickstart/hugegraph-client.md [hunk 3]
```diff
@@ -4,37 +4,35 @@ linkTitle: "Develop with HugeGraph-Client"
 weight: 6
 ---
 
-### 1 HugeGraph-Client概述
+### 1 Overview Of Hugegraph
 
-HugeGraph-Client向HugeGraph-Server发出HTTP请求，获取并解析Server的执行结果。目前仅提供了Java版，用户可以使用HugeGraph-Client编写Java代码操作HugeGraph，比如元数据和图数据的增删改查，或者执行gremlin语句。
+HugeGraph-Client sends HTTP request to HugeGraph-Server to obtain and parse the execution result of Server. Currently only the Java version is provided. You can use HugeGraph-Client to write Java code to operate HugeGraph, such as adding, deleting, modifying, and querying metadata and graph data, or executing gremlin statements.
```
Comment (javeme) [inline content/en/docs/quickstart/hugegraph-client.md:9]: Currently only the HugeGraph-Client for Java is provided.
querying schema and graph data...

File: content/en/docs/quickstart/hugegraph-client.md [hunk 4]
```diff
@@ -340,11 +338,11 @@ public class BatchExample {
...
+### 4.4 Run The Example
 
-运行Example之前需要启动Server, 启动过程见[HugeGraph-Server Quick Start](/docs/quickstart/hugegraph-server)
+Before running Example, you need to start the Server. For the startup process, see[HugeGraph-Server Quick Start](/docs/quickstart/hugegraph-server)
 
-### 4.5 Example示例说明
+### 4.5 More Information About Example
 
-示例说明见[HugeGraph-Client基本API介绍](/docs/clients/hugegraph-client)
+See[Introduce basic api of HugeGraph-Client](/docs/clients/hugegraph-client)
```
Comment (javeme) [inline content/en/docs/quickstart/hugegraph-client.md:347]: api => API

**Review by javeme — COMMENTED** [review]
Thanks for your contribution, some small comments

File: content/en/docs/quickstart/hugegraph-client.md (hunk 3 above)
Comment (Alita-Nan) [reply content/en/docs/quickstart/hugegraph-client.md]: Sorry, I'm not sure what the second sentence means, so I decided to follow the result of google translate. As for the first one, I think we both mean the same thing🤣?

File: content/en/docs/quickstart/hugegraph-client.md (hunk 4 above)
Comment (Alita-Nan) [reply content/en/docs/quickstart/hugegraph-client.md:347]: I have fixed them all.

Comment (javeme) [reply content/en/docs/quickstart/hugegraph-client.md:347]: Did not see the changes, did you forget to submit the latest code?

File: content/en/docs/quickstart/hugegraph-client.md (hunk 3 above)
Comment (javeme) [reply content/en/docs/quickstart/hugegraph-client.md]: We can adjust some statements:
`Currently only the Java version is provided` => `Currently only the HugeGraph-Client for Java is provided`
`querying metadata and graph data` => `querying schema and graph data...`

File: content/en/docs/quickstart/hugegraph-client.md (hunk 4 above)
Comment (imbajin) [reply content/en/docs/quickstart/hugegraph-client.md:347]: you could check the commit in github [files](https://github.com/apache/incubator-hugegraph-doc/pull/116/files)

and use github app it easier

**Review by javeme — APPROVED** [review]

File: content/en/docs/quickstart/hugegraph-client.md (hunk 4 above)
Comment (Alita-Nan) [reply content/en/docs/quickstart/hugegraph-client.md]: What about now? 😥 Sorry for that mistake.

**Pull request merged**